tool/tbot/anonymous_telemetry.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"context"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/bufbuild/connect-go"
+	"github.com/google/uuid"
+	"github.com/gravitational/trace"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/gen/proto/go/prehog/v1alpha"
+	"github.com/gravitational/teleport/gen/proto/go/prehog/v1alpha/v1alphaconnect"
```

Comment:
I've been using these aliases whenever possible.

```suggestion
	prehogv1 "github.com/gravitational/teleport/gen/proto/go/prehog/v1alpha"
	prehogv1c "github.com/gravitational/teleport/gen/proto/go/prehog/v1alpha/v1alphaconnect"
```